functions/go/set-default-labels/README.md

```diff
@@ -0,0 +1,54 @@
+# set-default-labels
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `set-default-labels` function applies KPT label convention to your KPT package.
+
+
+### `app.kubernetes.io/name` 
+
+Running the function can add this label using the value from the root `Kptfile` name.
+
+This convention believes that all KRM resources under the same kpt package should be served for a specific application.
+An application (or KPT package) can be composed by other applications (nested sub KPT packages). 
+So the function can accept multiple Kptfile and use the root Kptfile to set the app name.
+
+<!--mdtogo-->
+
+You can learn more about recommended labels [here][recommended labels].
+
+## Usage
+
+This function should be run in a KPT package. It does not require function config.
+
+### Run the function once
+```bash
+kpt fn eval --image set-default-labels
```

Comment:
Should have an image tag here.

Also please add `$` at the beginning. IIRC the copy button use it to determine which part should be copied.
Please audit all the README files in this PR.

Reply:
Thanks! This is helpful 

> Should have an image tag here.
> 
> IIRC the copy button use it to determine which part should be copied. Please audit all the README files in this PR.

Thank you. This is really helpful.